### ISSUE TITLE
Add need changing LVResizableNavigationBar as well

### ISSUE THREAD

**Issue by OthmanT** [Levelmoney/resizable-navigation-bar#1]
In the how to section, you should add that we need to change the NavigationController class AND the bar within.

Thanks for the good job.

**Comment by tdansel88** [comment]
You're right - but only if you're using storyboards or using initWithNavigationBarClass:toolbarClass.  Updated README with caveat
